apps/prairielearn/src/components/QuestionsTable.html.tsx

```diff
@@ -316,8 +316,11 @@ function CreateQuestionModal({
           required
           aria-describedby="start_from_help"
         >
-          <option value="Empty question">Empty question</option>
-          <option value="Template">Template</option>
+          <option value="empty">Empty question</option>
+          <option value="example">Default Template</option>
```

Comment:
Nits:

- Consistent capitalization ("Default template" to match "Empty question")
- Consider something like "Built-in template"? "Default" has strange connotations to me in this context that I can't quite put my finger on.

Reply:
The last three feel weird to me. "system" seems too technical? "standard" is probably the best of these, but still is maybe a bit off. "Starter templates"?

We could drop "template" entirely from this option? e.g. "Common question types"?

Reply:
"PrairieLearn templates" isn't bad. Maybe post this on Slack for somre more bickshedding? 😅